marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-vis/graph-vis.tsx

```diff
@@ -205,6 +247,26 @@ const GraphVis: React.FC<Props> = (props) => {
         props.handleEntitySelection(nodes[0]);
       }
     },
+    click: (event) => {
+      //if click is on an edge
+      if (event.edges.length > 0 && event.nodes.length < 1) {
+        let connectedNodes = window.graphVisApi.getConnectedNodes(event.edges[0]);
```

Comment:
We should directly refer to the network function here like **network.getConnectedNodes(event.edges[0])** instead of adding it in window object first and then calling it from there. This will also improve the performance of the graph edge modal display.